lambdas/eventbrite/serverless.yml

```diff
@@ -0,0 +1,28 @@
+service: pull-eventbrite-events
+provider:
+  name: aws
+  runtime: nodejs8.10
+  iamRoleStatements:
+    - Effect: Allow
+      Action:
+        - s3:*
+      Resource:
+        Fn::Join:
+          - ""
+          - - "arn:aws:s3:::"
+            - "Ref" : "EventbriteEventsBucket"
+            - "/*"
+functions:
+  hello:
+    handler: handler.hello
+    events:
+      - schedule: rate(1 hour)
+    environment:
+      TZ: Europe/Belfast
+      EVENTBRITE_API_TOKEN: ${ssm:eventbriteApiToken~true}
```

Comment:
How are you configuring the `EVENTBRITE_API_TOKEN` for the functions they get deployed to AWS? Does serverless handle this?

Reply:
@apoclyps Yea, as long as you set them in the Systems Manager Parameter store, serverless seems to take care of pull them in for you - https://serverless.com/blog/serverless-secrets-api-keys/#handling-secrets-for-small-teams--projects
There's some details in the updated readme file in this PR.